app/src/main/java/com/mk/tasky/agenda/detail/reminder/presentation/DetailReminderState.kt

```diff
@@ -1,13 +1,17 @@
 package com.mk.tasky.agenda.detail.reminder.presentation
 
 import com.mk.tasky.agenda.detail.components.model.ReminderTypes
+import com.mk.tasky.agenda.detail.reminder.presentation.model.DetailRegistrationInformation
 import java.time.LocalDateTime
 
 data class DetailReminderState(
     val isEditing: Boolean = true,
-    val date: LocalDateTime = LocalDateTime.now(),
-    val title: String = "New reminder",
-    val description: String = "Description",
-    val showDropdown: Boolean = false,
-    val selectedReminder: ReminderTypes = ReminderTypes.THIRTY_MINUTES
+    val information: DetailRegistrationInformation = DetailRegistrationInformation(
```

Comment:
To me it's not fully clear what this class should represent, considering there's a copy of it called `editableInformation` 🤔 
Is this used to minimize duplicate code between events, tasks and reminders or what is the goal? I find the name a bit confusing

Reply:
This is used because if a user edits Title, Description, the date of the reminder, or the time before the notification, but doesn't click save, then it will restore the information to before editing

I just realized I named it "DetailRegistrationInformation" and not "ReminderInformation"